tests/ci/kickoff_devicefarm_job.sh

```diff
@@ -74,11 +74,20 @@ function compile_for_android() {
   # |ANDROID_APK| and |ANDROID_TEST_APK| are apk names corresponding to the settings in the test runner gradle file.
   # The working directory of this script should under `tests/ci`. 
   # AWSLCAndroidTestRunner should be at `tests/ci/android/AWSLCAndroidTestRunner`.
-  cd android/AWSLCAndroidTestRunner
+  pushd android/AWSLCAndroidTestRunner
   export ANDROID_APK_LOCATION='android/AWSLCAndroidTestRunner/app/build/outputs/apk'
   if [[ "${FIPS}" = true ]]; then
-    # FIPS (Release Shared)
-    ./gradlew assembleDebug assembleAndroidTest -PFIPS --offline
+    if [[ "${SHARED}" = true ]]; then
+      # FIPS (Release Shared)
+      ./gradlew assembleDebug assembleAndroidTest -PFIPS -PShared --offline
+    else
+      # FIPS (Release Static), go dependencies need to be copied from AWS-LC.
+      mkdir app/src/main/cpp/util
+      cp -r ../../../../util/godeps.go app/src/main/cpp/util/godeps.go
```

Comment:
Just curious, why do go dependencies need to be copied for the static build?

Reply:
The static FIPS build runs a `delocate.go` program to parse and rewrite the assembly, in order to resolve some FIPS integrity check requirements. This makes golang a hard dependency for the static FIPS build. I'm not a go expert, but these files seem to be needed for any program written in golang since they define go dependencies the program needs(?)